docs/pages/kubernetes-access/troubleshooting.mdx

```diff
@@ -0,0 +1,214 @@
+---
+title: Kubernetes Access Troubleshooting
+description: Troubleshooting common issues with Kubernetes access
+---
+
+This guide describes common issues with Kubernetes Access and how to resolve them.
```

Comment:
This is super nit-picky, and we don't yet have this written into our style guide, but watch that not stop me 😄 

I would call this a "doc" instead of a "guide". I see a guide as a subset of documentation that specifically 'guides' the reader through a process. This is a collection of useful information, more akin to 'reference' material, but not explicitly so. Therefore: 

```suggestion
This page describes common issues with Kubernetes Access and how to resolve them.
```